android/src/main/java/com/rudderstack/android/RudderAnalytics.kt

```diff
@@ -147,21 +145,44 @@ private val infrastructurePlugins
     )
 private val messagePlugins
     get() = listOf(
-        PlatformInputsPlugin(),
-        ExtractStatePlugin(),
-        FillDefaultsPlugin(),
+        ReinstatePlugin(), PlatformInputsPlugin(), ExtractStatePlugin(), FillDefaultsPlugin(),
         SessionPlugin()
     )
 
 private fun Analytics.startup() {
     addPlugins()
     associateStates()
-    androidStorage.let {
-        contextState?.update(it.context)
-    }
+//    initializeSessionManagement()
+}
+
+/*private fun Analytics.reinstate() {
```

Comment:
Let's remove this code if not used anymore. I can see that Reinstate() is now a plugin 😄